client/api/omni/management/management.proto

```diff
@@ -91,6 +93,7 @@ message KubeconfigRequest {
   string service_account_user = 3;
   repeated string service_account_groups = 4;
   string grant_type = 5;
+  bool admin = 6;
```

Comment:
Does this mean "break-the-glass service account"? Isn't the feature only for emergency use cases? Using it in automation would be abuse of the feature, no?

Reply:
We can rename it, not sure about the new name though

Reply:
no idea :)

Reply:
Can call it break-glass

Reply:
I'm not super sure if we should change it, open for more opinions. @smira wdyt?

Reply:
`break_glass` sounds better to me, and probably we should do same for the CLI? 

as it's not 'admin', it's same level of access as via Omni

Reply:
updated to `break_glass` everywhere